test/infrastructure/docker/main.go

```diff
@@ -51,11 +51,9 @@ import (
 	expv1 "sigs.k8s.io/cluster-api/exp/api/v1beta1"
```

Comment:
* Please drop v1alpha3 from kustomization.yaml files
* Some more in controlplane/kubeadm/PROJECT, exp/addons/PROJECT, test/infrastructure/docker/exp/PROJECT

Reply:
Related Q: What are the `PROJECT` files for?

Reply:
Kubebuilder is storing the CRDs it generated there. No idea what its doing with that info. As we don't use kubebuilder anymore I'm pretty sure we can delete these files. Same for the scaffold markers we have in at least sone main.go files. Separate PR?

Reply:
Opened a PR here: https://github.com/kubernetes-sigs/cluster-api/pull/9344

BTW - when did we stop using kubebuilder? I was trying to search for something but didn't turn anything up.